lib/FrontendTool/FrontendTool.cpp

```diff
@@ -262,6 +263,185 @@ template <> struct ObjectTraits<LoadedModuleTraceFormat> {
 }
 }
 
+template <typename SetLike, typename Item>
+static bool contains(const SetLike &setLike, Item item) {
+  return setLike.find(item) != setLike.end();
+}
+
+/// Get a uniqued list of modules imported by \c module .
+///
+/// By default, all imports are included.
```

Comment:
https://www.spellchecker.net/misspellings/uniqued

```suggestion
/// Get a unique list of modules imported by \c module.
///
/// By default, all imports are included.
```

Reply:
Unique is technically considered an adjective (https://www.merriam-webster.com/dictionary/unique) but I've also seen it use as a verb, as in "unique the array". Just like "sorted list", I'm using "uniqued list". (Other jargon example: https://www.merriam-webster.com/dictionary/performant doesn't return any hits.)

That said, it's probably just simpler to replace "uniqued list" with "set" to skip the debate. :p